_src/app/habitat/Habitat.js

```diff
@@ -483,6 +505,68 @@ define([
 
                 return lang.mixin(this.transects[key], obj);
             }.bind(this));
+        },
+        showValidation(width) {
+            // summary:
+            //      shows validation messages
+            console.info('app/habitat/Habitat:showValidation', arguments);
+
+            const validation = this._validateWidth(width);
+
+            domClass.remove(this.validateMsg, 'alert-info alert-danger');
+            domClass.add(this.validateMsg, 'hidden');
+
+            let type = 'danger';
+            if (validation.result) {
+                type = 'info';
+            }
+
+            if (validation.message) {
+                domClass.add(this.validateMsg, `alert alert-${type}`);
+                domClass.remove(this.validateMsg, 'hidden');
+                this.validateMsg.innerHTML = validation.message;
+            } else {
+                domClass.add(this.validateMsg, 'hidden');
+            }
+        },
+        _validateWidth(wettedWidth) {
+            // summary:
+            //      vlidation function
+            // boolean
+            console.info('app/habitat/Habitat:_validateWidth', arguments);
+
+            if (!wettedWidth) {
+                let message = null;
+
+                const hasStartDistances = this.store.data.some(row => {
+                    return row[FN.transectMeasurements.DISTANCE_START];
+                });
+
+                if (hasStartDistances) {
+                    message = 'Wetted Width does not have a value. Remember that the ' +
+                             'distance from starting bank needs to be less than the wetted width.';
+                }
+
+                return {
+                    result: true,
+                    message: message
+                };
+            }
+
+            const isValid = this.store.data.every(row => {
+                const start = row[FN.transectMeasurements.DISTANCE_START];
+
+                return !isNaN(start) && start < wettedWidth;
+            });
+
+            let message = isValid ?
+                null :
+                'The distance from the starting bank cannot be greater than the wetted width.';
```

Comment:
Might want to change this to "greater than _or equal to_".

Reply:
I think if  the width is equal to the start then you are at the other bank. I think less than makes more sense.

Reply:
You're right. SB must be less than WW would be even more clear.